components/application-operator/charts/application/templates/deployment.yaml

```diff
@@ -73,7 +73,7 @@ spec:
           runAsUser: {{ .Values.podSecurityPolicy.runAsUser }}
 {{- end }}
 ---
-{{- if not .Values.global.isBEBEnabled }}
+{{- if and (not .Values.global.isBEBEnabled) (not .Values.global.centralApplicationConnectivityValidatorEnabled) }}
```

Comment:
I think the condition should go to the top. Otherwise for standalone mode unused stuff will be created.